

<!DOCTYPE html>
<html class="writer-html5" lang="en" >
<head>
  <meta charset="utf-8" />
  
  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
  
  <title>Guide to ENA submission (webin) &mdash; Swedish Biodiversity Data Infrastructure: Molecular Data  documentation</title>
  

  
  <link rel="stylesheet" href="_static/css/theme.css" type="text/css" />
  <link rel="stylesheet" href="_static/pygments.css" type="text/css" />

  
  

  
  

  

  
  <!--[if lt IE 9]>
    <script src="_static/js/html5shiv.min.js"></script>
  <![endif]-->
  
    
      <script type="text/javascript" id="documentation_options" data-url_root="./" src="_static/documentation_options.js"></script>
        <script src="_static/jquery.js"></script>
        <script src="_static/underscore.js"></script>
        <script src="_static/doctools.js"></script>
    
    <script type="text/javascript" src="_static/js/theme.js"></script>

    
    <link rel="index" title="Index" href="genindex.html" />
    <link rel="search" title="Search" href="search.html" />
    <link rel="next" title="Taxonomic annotation of ASVs" href="std-taxon-annotation.html" />
    <link rel="prev" title="Molecular biodiversity data in SBDI" href="index.html" />
<link rel="stylesheet" type="text/css"
     href="_static/custom.css" />


</head>

<body class="wy-body-for-nav">

   
  <div class="wy-grid-for-nav">
    
    <nav data-toggle="wy-nav-shift" class="wy-nav-side">
      <div class="wy-side-scroll">
        <div class="wy-side-nav-search" >
          

          
            <a href="index.html" class="icon icon-home"> Swedish Biodiversity Data Infrastructure: Molecular Data
          

          
          </a>

          
            
            
          

          
<div role="search">
  <form id="rtd-search-form" class="wy-form" action="search.html" method="get">
    <input type="text" name="q" placeholder="Search docs" />
    <input type="hidden" name="check_keywords" value="yes" />
    <input type="hidden" name="area" value="default" />
  </form>
</div>

          
        </div>

        
        <div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="main navigation">
          
            
            
              
            
            
              <ul class="current">
<li class="toctree-l1 current"><a class="current reference internal" href="#">Guide to ENA submission (webin)</a><ul>
<li class="toctree-l2"><a class="reference internal" href="#preparation-for-submission">Preparation for submission</a><ul>
<li class="toctree-l3"><a class="reference internal" href="#step-1-prepare-data-and-metadata">Step 1: Prepare data and metadata</a></li>
<li class="toctree-l3"><a class="reference internal" href="#step-2-register-with-ena">Step 2: Register with ENA</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="#interactive-submission">Interactive submission</a><ul>
<li class="toctree-l3"><a class="reference internal" href="#step-1-log-in-to-submission-portal">Step 1: Log in to submission portal</a></li>
<li class="toctree-l3"><a class="reference internal" href="#step-2-register-study">Step 2: Register study</a></li>
<li class="toctree-l3"><a class="reference internal" href="#step-3-register-samples">Step 3: Register samples</a><ul>
<li class="toctree-l4"><a class="reference internal" href="#step-3a-select-sample-checklist">Step 3a: Select sample checklist</a></li>
<li class="toctree-l4"><a class="reference internal" href="#step-3b-add-sample-attributes">Step 3b: Add sample attributes</a></li>
<li class="toctree-l4"><a class="reference internal" href="#step-3c-download-spreadsheet-template">Step 3c: Download spreadsheet template</a></li>
<li class="toctree-l4"><a class="reference internal" href="#step-3d-edit-spreadsheet-structure">Step 3d: Edit spreadsheet structure</a></li>
<li class="toctree-l4"><a class="reference internal" href="#step-3e-add-sample-metadata">Step 3e: Add sample metadata</a></li>
<li class="toctree-l4"><a class="reference internal" href="#step-3f-upload-spreadsheet">Step 3f: Upload spreadsheet</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="#step-4-prepare-and-upload-read-files">Step 4: Prepare and upload read files</a><ul>
<li class="toctree-l4"><a class="reference internal" href="#step-4a-compress-files-and-calculate-checksums">Step 4a: Compress files and calculate checksums</a></li>
<li class="toctree-l4"><a class="reference internal" href="#step-4b-upload-read-files-to-ena">Step 4b: Upload read files to ENA</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="#step-5-submit-sequence-reads">Step 5: Submit sequence reads</a><ul>
<li class="toctree-l4"><a class="reference internal" href="#step-5a-download-the-experiment-run-template">Step 5a: Download the experiment/run template</a></li>
<li class="toctree-l4"><a class="reference internal" href="#step-5b-describe-experiment-and-runs">Step 5b: Describe experiment and runs</a></li>
</ul>
</li>
<li class="toctree-l3"><a class="reference internal" href="#step-6-submit-to-production-service">Step 6: Submit to production service</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="#post-submission-editing">Post-submission editing</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="std-taxon-annotation.html">Taxonomic annotation of ASVs</a></li>
</ul>

            
          
        </div>
        
      </div>
    </nav>

    <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap">

      
      <nav class="wy-nav-top" aria-label="top navigation">
        
          <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
          <a href="index.html">Swedish Biodiversity Data Infrastructure: Molecular Data</a>
        
      </nav>


      <div class="wy-nav-content">
        
        <div class="rst-content">
        
          

















<div role="navigation" aria-label="breadcrumbs navigation">

  <ul class="wy-breadcrumbs">
    
      <li><a href="index.html" class="icon icon-home"></a> &raquo;</li>
        
      <li>Guide to ENA submission (webin)</li>
    
    
      <li class="wy-breadcrumbs-aside">
        
          
            <a href="_sources/ena-metabar.rst.txt" rel="nofollow"> View page source</a>
          
        
      </li>
    
  </ul>

  
  <hr/>
</div>
          <div role="main" class="document" itemscope="itemscope" itemtype="http://schema.org/Article">
           <div itemprop="articleBody">
            
  <div class="toctree-wrapper compound">
</div>
<p><em>Update Note (29 Oct 2021):
This is an adaptation to recent changes at ENA, using the Webin Portal for interactive submission.
Please send bug reports and/or suggestions to</em> <a class="reference external" href="https://docs.biodiversitydata.se/support/"><em>SBDI support</em></a>.</p>
<div class="section" id="guide-to-ena-submission-webin">
<h1>Guide to ENA submission (webin)<a class="headerlink" href="#guide-to-ena-submission-webin" title="Permalink to this headline">¶</a></h1>
<p>This is a guide on how to submit sequence reads from environmental samples to the <a class="reference external" href="https://www.ebi.ac.uk/ena/">European Nucleotide Archive (ENA)</a>, provided by the <a class="reference external" href="https://biodiversitydata.se/">Swedish Biodiversity Data Infrastructure (SBDI)</a>. Our guide is largely a summary of <a class="reference external" href="https://ena-docs.readthedocs.io/en/latest/index.html">ENA’s own extensive instructions</a>, with added pointers on issues specific to submission of metarbarcoding data, as well as on more general matters that may confuse first-time contributors. While ENA provides <a class="reference external" href="https://ena-docs.readthedocs.io/en/latest/general-guide.html">three different routes for submission</a>, we describe interactive submission via the Webin portal here.</p>
<div class="section" id="preparation-for-submission">
<h2>Preparation for submission<a class="headerlink" href="#preparation-for-submission" title="Permalink to this headline">¶</a></h2>
<div class="section" id="step-1-prepare-data-and-metadata">
<h3>Step 1: Prepare data and metadata<a class="headerlink" href="#step-1-prepare-data-and-metadata" title="Permalink to this headline">¶</a></h3>
<p>In ENA, raw sequencing output from a next generation platform, including e.g. base calls and per-base quality scores, is accepted in <a class="reference external" href="https://ena-docs.readthedocs.io/en/latest/fileprep/reads.html">FASTQ, CRAM or BAM format</a>. Before submission, make sure that sequencing adapters have been removed (<em>trimmed</em>), and that reads have been assigned to their sample of origin (<em>demultiplexed</em>). In addition, gather all the information (<em>metadata</em>) you have about how, when and where you acquired the samples and generated the reads, as well as any contextual (environmental or clinical) data that was collected during sampling (see <a class="reference external" href="https://ena-docs.readthedocs.io/en/latest/general-guide/metadata.html">ENA’s metadata model</a>).</p>
</div>
<div class="section" id="step-2-register-with-ena">
<h3>Step 2: Register with ENA<a class="headerlink" href="#step-2-register-with-ena" title="Permalink to this headline">¶</a></h3>
<p>To be able to submit data to ENA, you need to register an account. Go to the <a class="reference external" href="https://wwwdev.ebi.ac.uk/ena/submit/webin/login">Webin submission Portal</a>, select <em>Register</em> to fill out the form, and save your account details. You will receive a confirmation email with your account name.</p>
</div>
</div>
<div class="section" id="interactive-submission">
<h2>Interactive submission<a class="headerlink" href="#interactive-submission" title="Permalink to this headline">¶</a></h2>
<div class="section" id="step-1-log-in-to-submission-portal">
<h3>Step 1: Log in to submission portal<a class="headerlink" href="#step-1-log-in-to-submission-portal" title="Permalink to this headline">¶</a></h3>
<p>ENA provides two submission services: one for <a class="reference external" href="https://wwwdev.ebi.ac.uk/ena/submit/webin/login">test submission</a> and one for (real) <a class="reference external" href="https://www.ebi.ac.uk/ena/submit/webin/login">production submission</a>. Make sure that your data pass validation in the test procedure, before submitting anything to the production environment. All test submissions are removed the following day, but you will save all metadata as tab-separated text (*.tsv) files that you can reuse later.</p>
<p>After login, you are directed to the Dashboard, which gives you an overview of all available submission activities and reports (i.e. lists of successfully submitted samples etc.). Via the top-left hamburger icon (<img alt="hamburger-icon" src="_images/hamburger-icon.png" />) you can either go back to this Dashboard, or jump directly to some activity or report.</p>
</div>
<div class="section" id="step-2-register-study">
<h3>Step 2: Register study<a class="headerlink" href="#step-2-register-study" title="Permalink to this headline">¶</a></h3>
<p>The <em>study (project)</em> object is linked to samples and sequence reads via experiments, and is typically what you cite in publications. Click <em>Studies (Projects) | Register Study</em> to start filling out the Study form. Note that a single release date is set for all data within a study, and that you thus may want to split sequenced batches of samples into multiple ENA studies. After entering the required metadata for a study, click <em>Submit | OK</em>. If successful, you will receive a confirmation message, and should be able to see your study listed in the <em>Studies (Projects) | Study Report</em> page.</p>
</div>
<div class="section" id="step-3-register-samples">
<h3>Step 3: Register samples<a class="headerlink" href="#step-3-register-samples" title="Permalink to this headline">¶</a></h3>
<p><em>Samples</em> are the source material from which your sequences derive, and the searchability and usability of your submitted data will depend on how well you document these samples. Go to <em>Samples | Register Samples</em> and click <em>Download spreadsheet to register samples</em> to start the process.</p>
<div class="section" id="step-3a-select-sample-checklist">
<h4>Step 3a: Select sample checklist<a class="headerlink" href="#step-3a-select-sample-checklist" title="Permalink to this headline">¶</a></h4>
<p>The <a class="reference external" href="https://www.ebi.ac.uk/ena/submit/checklists">ENA sample checklists</a> are partly overlapping sets of attributes (or data fields) that can be used to describe samples, and by selecting one of these you enable your sample metadata to be validated for correctness during submission. For environmental and organismal (host-associated) samples, alike, we recommend using one of the <em>Environmental Checklists</em> and, among these, to select the alternative from the <em>Genomic Standards Consortium (GSC) MIxS checklists</em> that provides the most specific match to your sampled environment, for example:</p>
<table class="docutils align-default">
<colgroup>
<col style="width: 50%" />
<col style="width: 50%" />
</colgroup>
<thead>
<tr class="row-odd"><th class="head"><p>Sampled environment</p></th>
<th class="head"><p>Recommended checklist</p></th>
</tr>
</thead>
<tbody>
<tr class="row-even"><td><p>Air or general, above-ground, terrestrial</p></td>
<td><p>GSC MixS air</p></td>
</tr>
<tr class="row-odd"><td><p>Epi- or endophytic (e.g. leaf, root)</p></td>
<td><p>GSC MIxS plant associated</p></td>
</tr>
<tr class="row-even"><td><p>Epi- or endozoic (e.g. spider gut, animal skin)</p></td>
<td><p>GSC MIxS host associated</p></td>
</tr>
<tr class="row-odd"><td><p>Fresh- or seawater</p></td>
<td><p>GSC MixS water</p></td>
</tr>
<tr class="row-even"><td><p>Human gut / oral / skin / vaginal</p></td>
<td><p>GSC MIxS human gut / oral / skin / vaginal</p></td>
</tr>
<tr class="row-odd"><td><p>Human non- gut / oral / skin / vaginal</p></td>
<td><p>GSC MIxS human associated</p></td>
</tr>
<tr class="row-even"><td><p>Sediment</p></td>
<td><p>GSC MixS sediment</p></td>
</tr>
<tr class="row-odd"><td><p>Soil</p></td>
<td><p>GSC MixS soil</p></td>
</tr>
</tbody>
</table>
<p>Note that most GSC MIxS checklists have similar setups of mandatory and recommended attributes, i.e. differ mainly in terms of which optional attributes can be added and validated during submission. The environmental attributes <em>altitude, elevation</em> and <em>depth</em>, are mandatory only for some lists, however. Furthermore, the GSC MIxS built environment list (not mentioned above) has several unique mandatories.</p>
</div>
<div class="section" id="step-3b-add-sample-attributes">
<h4>Step 3b: Add sample attributes<a class="headerlink" href="#step-3b-add-sample-attributes" title="Permalink to this headline">¶</a></h4>
<p>Once you have selected a checklist, you can inspect the attributes available in that list. All <em>Mandatory Fields</em> are preselected, and you can add <em>Optional Fields</em> by ticking their boxes. We suggest that you, at least, tick the following optional attributes for metabarcoding data*:</p>
<table class="docutils align-default">
<colgroup>
<col style="width: 50%" />
<col style="width: 50%" />
</colgroup>
<thead>
<tr class="row-odd"><th class="head"><p>Checklist</p></th>
<th class="head"><p>Optional attribute</p></th>
</tr>
</thead>
<tbody>
<tr class="row-even"><td><p>[ All ]</p></td>
<td><p>target gene</p></td>
</tr>
<tr class="row-odd"><td><p>[ All ]</p></td>
<td><p>target subfragment</p></td>
</tr>
<tr class="row-even"><td><p>[ All ]</p></td>
<td><p>pcr primers</p></td>
</tr>
<tr class="row-odd"><td><p>[ All ]</p></td>
<td><p>sequencing method</p></td>
</tr>
<tr class="row-even"><td><p>GSC MIxS host / plant associated</p></td>
<td><p>host taxid</p></td>
</tr>
</tbody>
</table>
<p>* <em>Disclaimer</em>: While target gene/subfragment and primers are typically associated with the (sequencing) experiment, ENA only lists these fields as optional at sample level. When queried about this, ENA suggests adding the fields to experiments through post-submission XML editing, but we consider this process too complex. Therefore, we still recommend adding them to your samples as outlined above. Please note, however, that if you’ve sequenced multiple targets (e.g., both 16S and 18S) using the same <em>physical</em> samples, you will then need to submit two sets of <em>digital</em> samples. If you choose this approach, please make it easy for data users to identify sequences originating from the same physical sample by adding sample aliases, such as <em>[physical_sample_id]_16S</em> and <em>[physical_sample_id]_18S</em>.</p>
<p>To ensure that metadata are validated and searchable, you should use existing attributes whenever possible, but you can also add custom attributes to describe your data, if needed (see <em>Add custom field</em> in top left corner of page. Write your field name <em>before</em> hitting the plus button). Adding more than a few of these is easier to do in a spreadsheet, though, and we explain how to do that in the next step.</p>
</div>
<div class="section" id="step-3c-download-spreadsheet-template">
<h4>Step 3c: Download spreadsheet template<a class="headerlink" href="#step-3c-download-spreadsheet-template" title="Permalink to this headline">¶</a></h4>
<p>After selecting optional attributes, click <em>Next</em>, and then the <em>Download TSV Template</em> button to download a tab-separated values (*.tsv) file. Open the file in your spreadsheet application of choice (In MS Excel, click <em>Data | Text to Columns | Delimited | Delimiter: Tab</em>, to separate text into columns, if needed). With the added optional attributes from 3b, a template created from the <em>GSC MIxS water</em> checklist should look like this:</p>
<table class="docutils align-default">
<colgroup>
<col style="width: 6%" />
<col style="width: 6%" />
<col style="width: 6%" />
<col style="width: 6%" />
<col style="width: 6%" />
<col style="width: 6%" />
<col style="width: 6%" />
<col style="width: 6%" />
<col style="width: 6%" />
<col style="width: 6%" />
<col style="width: 6%" />
<col style="width: 6%" />
<col style="width: 6%" />
<col style="width: 6%" />
<col style="width: 6%" />
<col style="width: 6%" />
<col style="width: 6%" />
<col style="width: 6%" />
</colgroup>
<tbody>
<tr class="row-odd"><td><p>Checklist</p></td>
<td><p>ERC000024</p></td>
<td><p>GSC MIxS water</p></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr class="row-even"><td><p>tax_id</p></td>
<td><p>scientific_name</p></td>
<td><p>sample_alias</p></td>
<td><p>sample_title</p></td>
<td><p>sample_description</p></td>
<td><p>project name</p></td>
<td><p>target gene</p></td>
<td><p>target subfragment</p></td>
<td><p>pcr primers</p></td>
<td><p>sequencing method</p></td>
<td><p>collection date</p></td>
<td><p>geographic location (country and/or sea)</p></td>
<td><p>geographic location (latitude)</p></td>
<td><p>geographic location (longitude)</p></td>
<td><p>depth</p></td>
<td><p>broad-scale environmental context</p></td>
<td><p>local environmental context</p></td>
<td><p>environmental medium</p></td>
</tr>
<tr class="row-odd"><td><p>#units</p></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td><p>DD</p></td>
<td><p>DD</p></td>
<td><p>m</p></td>
<td></td>
<td></td>
<td></td>
</tr>
</tbody>
</table>
</div>
<div class="section" id="step-3d-edit-spreadsheet-structure">
<h4>Step 3d: Edit spreadsheet structure<a class="headerlink" href="#step-3d-edit-spreadsheet-structure" title="Permalink to this headline">¶</a></h4>
<p>In the downloaded spreadsheet, you can add custom attributes (with units, where applicable) to the right of existing columns, and samples below the row starting with <em>#units</em>. For instance, if you add two custom attributes (<em>salinity [psu]</em> and <em>sampling_station_id</em>), and three samples (<em>xyz:1:01</em>, <em>xyz:1:02</em> and <em>xyz:1:03</em>) to the template from Step 3c, your spreadsheet structure should look like this:</p>
<table class="docutils align-default">
<colgroup>
<col style="width: 5%" />
<col style="width: 5%" />
<col style="width: 5%" />
<col style="width: 5%" />
<col style="width: 5%" />
<col style="width: 5%" />
<col style="width: 5%" />
<col style="width: 5%" />
<col style="width: 5%" />
<col style="width: 5%" />
<col style="width: 5%" />
<col style="width: 5%" />
<col style="width: 5%" />
<col style="width: 5%" />
<col style="width: 5%" />
<col style="width: 5%" />
<col style="width: 5%" />
<col style="width: 5%" />
<col style="width: 5%" />
<col style="width: 5%" />
</colgroup>
<tbody>
<tr class="row-odd"><td><p>Checklist</p></td>
<td><p>ERC000024</p></td>
<td><p>GSC MIxS water</p></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr class="row-even"><td><p>tax_id</p></td>
<td><p>scientific_name</p></td>
<td><p>sample_alias</p></td>
<td><p>sample_title</p></td>
<td><p>sample_description</p></td>
<td><p>project name</p></td>
<td><p>target gene</p></td>
<td><p>target subfragment</p></td>
<td><p>pcr primers</p></td>
<td><p>sequencing method</p></td>
<td><p>collection date</p></td>
<td><p>geographic location (country and/or sea)</p></td>
<td><p>geographic location (latitude)</p></td>
<td><p>geographic location (longitude)</p></td>
<td><p>depth</p></td>
<td><p>broad-scale environmental context</p></td>
<td><p>local environmental context</p></td>
<td><p>environmental medium</p></td>
<td><p>salinity</p></td>
<td><p>sampling_station_id</p></td>
</tr>
<tr class="row-odd"><td><p>#units</p></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td><p>DD</p></td>
<td><p>DD</p></td>
<td><p>m</p></td>
<td></td>
<td></td>
<td></td>
<td><p>psu</p></td>
<td></td>
</tr>
<tr class="row-even"><td></td>
<td></td>
<td><p>xyz:1:01</p></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr class="row-odd"><td></td>
<td></td>
<td><p>xyz:1:02</p></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr class="row-even"><td></td>
<td></td>
<td><p>xyz:1:03</p></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
</tbody>
</table>
<p>Remember to enter sample aliases that correspond to what you use in related publications. This will enable readers to find sample-specific metadata and read files, even if you only state a Project accession number in your paper. Sample aliases can, furthermore, be optionally displayed as <em>Unique name</em> in the <em>Sample report</em>.</p>
</div>
<div class="section" id="step-3e-add-sample-metadata">
<h4>Step 3e: Add sample metadata<a class="headerlink" href="#step-3e-add-sample-metadata" title="Permalink to this headline">¶</a></h4>
<p>Before adding actual sample metadata to your spreadsheet, take a close look at ENA’s explanations of selected attributes and lists of permitted values. These are available in the <a class="reference external" href="https://www.ebi.ac.uk/ena/browser/checklists">Sample Checklists browser</a>. Also note the following:</p>
<ul class="simple">
<li><p><strong>Taxon attributes for metabarcoding samples may be confusing</strong>. In this context, the <em>tax_id</em> &amp; <em>scientific_name</em> attributes do not typically refer to <em>sequenced</em> organisms, but rather describe <em>sampled</em> organisms or environments. The <em>scientific_name</em> value <em>spider metagenome</em> is, for example, used to describe samples from a spider or spider body part, i.e. not samples from which you have derived spider sequences. The attributes <em>tax_id</em> &amp; <em>scientific_name</em> should thus be selected from the list of <a class="reference external" href="https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?id=408169">environmental and organismal metagenomes in NCBI’s taxonomy browser</a>. For host-associated samples, also differentiate between these generic attributes (i.e. <em>tax_id</em> &amp; <em>scientific_name</em>) and <em>host taxid</em>, which you can also search for in <a class="reference external" href="https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi">NCBI’s taxonomy browser</a>, and should be as specific as possible.</p></li>
</ul>
<ul>
<li><p><strong>Some attributes should be selected from ontologies</strong>. To increase searchability, some attribute values should be selected from designated ontologies, which are formal specifications of terms used in certain contexts, and of how these terms relate to each other. You can browse or search the latest versions of ontologies used in ENA submission using the <a class="reference external" href="https://www.ebi.ac.uk/ols/index">EMBL-EBI Ontology Lookup Service (OLS)</a>. You can also use the following direct links as <em>starting points</em> for finding valid terms for some mandatory or recommended attributes in a GSC MixS checklists:</p>
<table class="docutils align-default">
<colgroup>
<col style="width: 33%" />
<col style="width: 33%" />
<col style="width: 33%" />
</colgroup>
<thead>
<tr class="row-odd"><th class="head"><p>Checklist</p></th>
<th class="head"><p>Ontology-linked attribute</p></th>
<th class="head"><p>ENA description</p></th>
</tr>
</thead>
<tbody>
<tr class="row-even"><td><p>[ All ]</p></td>
<td><p><a class="reference external" href="https://www.ebi.ac.uk/ols/ontologies/envo/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FENVO_00000428">broad-scale environmental context</a></p></td>
<td><p>Report the major environmental system the sample or specimen came from. The system(s) identified should have a coarse spatial grain, to provide the general environmental context of where the sampling was done (e.g. in the desert or a rainforest). We recommend using subclasses of EnvO’s biome class: <a class="reference external" href="http://purl.obolibrary.org/obo/ENVO_00000428">http://purl.obolibrary.org/obo/ENVO_00000428</a>. EnvO documentation about how to use the field: <a class="reference external" href="https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS">https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS</a>.</p></td>
</tr>
<tr class="row-odd"><td><p>[ All ]</p></td>
<td><p><a class="reference external" href="https://www.ebi.ac.uk/ols/ontologies/envo/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FENVO_01000813">local environmental context</a></p></td>
<td><p>Report the entity or entities which are in the sample or specimen’s local vicinity and which you believe have significant causal influences on your sample or specimen. We recommend using EnvO terms which are of smaller spatial grain than your entry for ‘broad-scale environmental context’. Terms, such as anatomical sites, from other OBO Library ontologies which interoperate with EnvO (e.g. UBERON) are accepted in this field. EnvO documentation about how to use the field: <a class="reference external" href="https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS">https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS</a>.</p></td>
</tr>
<tr class="row-even"><td><p>[ GSC MIxS non host / plant associated ]</p></td>
<td><p><a class="reference external" href="https://www.ebi.ac.uk/ols/ontologies/envo/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FENVO_00010483">environmental medium</a></p></td>
<td><p>Report the environmental material(s) immediately surrounding the sample or specimen at the time of sampling. We recommend using subclasses of ‘environmental material’ (<a class="reference external" href="http://purl.obolibrary.org/obo/ENVO_00010483">http://purl.obolibrary.org/obo/ENVO_00010483</a>). EnvO documentation about how to use the field: <a class="reference external" href="https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS">https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS</a> . Terms from other OBO ontologies are permissible as long as they reference mass/volume nouns (e.g. air, water, blood) and not discrete, countable entities (e.g. a tree, a leaf, a table top).</p></td>
</tr>
<tr class="row-odd"><td><p>[ GSC MIxS host associated ]</p></td>
<td><p><a class="reference external" href="https://www.ebi.ac.uk/ols/ontologies/envo/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FUBERON_0000465">environmental medium (host)</a></p></td>
<td><p>Report the environmental material(s) immediately surrounding the sample or specimen at the time of sampling. We recommend using subclasses of ‘environmental material’ (<a class="reference external" href="http://purl.obolibrary.org/obo/ENVO_00010483">http://purl.obolibrary.org/obo/ENVO_00010483</a>). EnvO documentation about how to use the field: <a class="reference external" href="https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS">https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS</a> . Terms from other OBO ontologies are permissible as long as they reference mass/volume nouns (e.g. air, water, blood) and not discrete, countable entities (e.g. a tree, a leaf, a table top).</p></td>
</tr>
<tr class="row-even"><td><p>[ GSC MIxS plant associated ]</p></td>
<td><p><a class="reference external" href="https://www.ebi.ac.uk/ols/ontologies/envo/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FPO_0025131">environmental medium (plant)</a></p></td>
<td><p>Report the environmental material(s) immediately surrounding the sample or specimen at the time of sampling. We recommend using subclasses of ‘environmental material’ (<a class="reference external" href="http://purl.obolibrary.org/obo/ENVO_00010483">http://purl.obolibrary.org/obo/ENVO_00010483</a>). EnvO documentation about how to use the field: <a class="reference external" href="https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS">https://github.com/EnvironmentOntology/envo/wiki/Using-ENVO-with-MIxS</a> . Terms from other OBO ontologies are permissible as long as they reference mass/volume nouns (e.g. air, water, blood) and not discrete, countable entities (e.g. a tree, a leaf, a table top).</p></td>
</tr>
<tr class="row-odd"><td><p>[ All ]</p></td>
<td><p><a class="reference external" href="http://purl.obolibrary.org/obo/OBI_0400103">sequencing method</a></p></td>
<td><p>Sequencing machine used. Where possible the term should be taken from the OBI list of DNA sequencers (<a class="reference external" href="http://purl.obolibrary.org/obo/OBI_0400103">http://purl.obolibrary.org/obo/OBI_0400103</a>).</p></td>
</tr>
</tbody>
</table>
<p>In the linked ontology tree views, click the plus sign next to a highlighted branch to show all instances of that term, and continue downwards until you find the most specific term that accurately describes your data. It is good practice to then register the term together with ontology acronym and accession, e.g: <em>marine pelagic biome (ENVO:01000023)</em>.</p>
</li>
</ul>
<ul class="simple">
<li><p><strong>Environmental attributes of host-associated samples are ambiguous</strong>. For instance, a spider may, in some sense, be the environment from which a host-associated sample derives. But as the external environment also may be of interest here, we tentatively suggest that you interpret <em>broad-scale</em> and <em>local environmental context</em> the same way as for non-host-associated samples, and use the most specific instance of <em>material anatomical entity (UBERON:0000465)</em> or <em>plant anatomical entity (PO:0025131)</em> for the <em>environmental medium</em> attribute of host- and plant associated samples, respectively.</p></li>
</ul>
<ul>
<li><p><strong>Geographic positions should be given in decimal degrees (DD)</strong>. If conversion is needed, use an online tool, such as the <a class="reference external" href="https://www.pgc.umn.edu/apps/convert/">PGC Coordinate Converter</a>, or the following formula:</p>
<table class="docutils align-default">
<colgroup>
<col style="width: 33%" />
<col style="width: 5%" />
<col style="width: 62%" />
</colgroup>
<tbody>
<tr class="row-odd"><td><p>Decimal Degrees</p></td>
<td><p>=</p></td>
<td><p>Degrees + Minutes/60 + Seconds/3600</p></td>
</tr>
<tr class="row-even"><td colspan="3"><p><em>Ex: 58°11’12.34’’ = 58 + 11/60 + 12.34/3600 = 58.1868°</em></p></td>
</tr>
<tr class="row-odd"><td colspan="3"><p><em>Ex: 58°11.21’ = 58 + 11.21/60 = 58.1868°</em></p></td>
</tr>
</tbody>
</table>
<p>The related <em>Geographic location (country and/or sea)</em> attribute should be selected from the <a class="reference external" href="https://www.insdc.org/submitting-standards/country-qualifier-vocabulary/">INSDC list for countries and seas</a>.</p>
</li>
</ul>
</div>
<div class="section" id="step-3f-upload-spreadsheet">
<h4>Step 3f: Upload spreadsheet<a class="headerlink" href="#step-3f-upload-spreadsheet" title="Permalink to this headline">¶</a></h4>
<p>With data added, your spreadsheet should look similar to this (admittedly, we have added a manufacturer name to the sequencing method ontology term, as it seemed to be missing from this specific device):</p>
<table class="docutils align-default">
<colgroup>
<col style="width: 5%" />
<col style="width: 5%" />
<col style="width: 5%" />
<col style="width: 5%" />
<col style="width: 5%" />
<col style="width: 5%" />
<col style="width: 5%" />
<col style="width: 5%" />
<col style="width: 5%" />
<col style="width: 5%" />
<col style="width: 5%" />
<col style="width: 5%" />
<col style="width: 5%" />
<col style="width: 5%" />
<col style="width: 5%" />
<col style="width: 5%" />
<col style="width: 5%" />
<col style="width: 5%" />
<col style="width: 5%" />
<col style="width: 5%" />
</colgroup>
<tbody>
<tr class="row-odd"><td><p>Checklist</p></td>
<td><p>ERC000024</p></td>
<td><p>GSC MIxS water</p></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr class="row-even"><td><p>tax_id</p></td>
<td><p>scientific_name</p></td>
<td><p>sample_alias</p></td>
<td><p>sample_title</p></td>
<td><p>sample_description</p></td>
<td><p>project name</p></td>
<td><p>target gene</p></td>
<td><p>target subfragment</p></td>
<td><p>pcr primers</p></td>
<td><p>sequencing method</p></td>
<td><p>collection date</p></td>
<td><p>geographic location (country and/or sea)</p></td>
<td><p>geographic location (latitude)</p></td>
<td><p>geographic location (longitude)</p></td>
<td><p>depth</p></td>
<td><p>broad-scale environmental context</p></td>
<td><p>local environmental context</p></td>
<td><p>environmental medium</p></td>
<td><p>salinity</p></td>
<td><p>sampling_station_id</p></td>
</tr>
<tr class="row-odd"><td><p>#units</p></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td><p>DD</p></td>
<td><p>DD</p></td>
<td><p>m</p></td>
<td></td>
<td></td>
<td></td>
<td><p>psu</p></td>
<td></td>
</tr>
<tr class="row-even"><td><p>1874687</p></td>
<td><p>marine plankton metagenome</p></td>
<td><p>xyz:1:01</p></td>
<td><p>Microbial plankton sample 1</p></td>
<td></td>
<td><p>Microbial plankton diversity along Baltic salinity gradient July 2018</p></td>
<td><p>16S rRNA</p></td>
<td><p>V3-V4</p></td>
<td><p>341F (CCTACGGGNGGCWGCAG) and 805R (GACTACHVGGGTATCTAATCC)</p></td>
<td><p>Illumina MiSeq</p></td>
<td><p>2018-07-23</p></td>
<td><p>Baltic Sea</p></td>
<td><p>61.407611</p></td>
<td><p>19.472809</p></td>
<td><p>3</p></td>
<td><p>marine pelagic biome (ENVO:01000023)</p></td>
<td><p>brackish water body (ENVO:01001321)</p></td>
<td><p>brackish water (ENVO:00002019)</p></td>
<td><p>3.13</p></td>
<td><p>A13</p></td>
</tr>
<tr class="row-odd"><td><p>1874687</p></td>
<td><p>marine plankton metagenome</p></td>
<td><p>xyz:1:02</p></td>
<td><p>Microbial plankton sample 2</p></td>
<td></td>
<td><p>Microbial plankton diversity along Baltic salinity gradient July 2018</p></td>
<td><p>16S rRNA</p></td>
<td><p>V3-V4</p></td>
<td><p>341F (CCTACGGGNGGCWGCAG) and 805R (GACTACHVGGGTATCTAATCC)</p></td>
<td><p>Illumina MiSeq</p></td>
<td><p>2018-07-24</p></td>
<td><p>Baltic Sea</p></td>
<td><p>62.805337</p></td>
<td><p>19.912262</p></td>
<td><p>3</p></td>
<td><p>marine pelagic biome (ENVO:01000023)</p></td>
<td><p>brackish water body (ENVO:01001321)</p></td>
<td><p>brackish water (ENVO:00002019)</p></td>
<td><p>5.64</p></td>
<td><p>A14</p></td>
</tr>
<tr class="row-even"><td><p>1874687</p></td>
<td><p>marine plankton metagenome</p></td>
<td><p>xyz:1:03</p></td>
<td><p>Microbial plankton sample 3</p></td>
<td></td>
<td><p>Microbial plankton diversity along Baltic salinity gradient July 2018</p></td>
<td><p>16S rRNA</p></td>
<td><p>V3-V4</p></td>
<td><p>341F (CCTACGGGNGGCWGCAG) and 805R (GACTACHVGGGTATCTAATCC)</p></td>
<td><p>Illumina MiSeq</p></td>
<td><p>2018-07-25</p></td>
<td><p>Baltic Sea</p></td>
<td><p>64.970396</p></td>
<td><p>23.076324</p></td>
<td><p>3</p></td>
<td><p>marine pelagic biome (ENVO:01000023)</p></td>
<td><p>brackish water body (ENVO:01001321)</p></td>
<td><p>brackish water (ENVO:00002019)</p></td>
<td><p>6.1</p></td>
<td><p>A15</p></td>
</tr>
</tbody>
</table>
<p>If so, go back to <em>Samples | Register Samples</em>, click <em>Upload filled spreadsheet to register samples</em>, select your edited *.tsv file, and click <em>Submit Completed Spreadsheet</em>. Given that your spreadsheet structure is correct, you will receive a confirmation message, and your samples should now be listed under <em>Samples | Samples Report</em>. Note that you need to click <em>Show unique name</em> to display sample aliases there.</p>
</div>
</div>
<div class="section" id="step-4-prepare-and-upload-read-files">
<h3>Step 4: Prepare and upload read files<a class="headerlink" href="#step-4-prepare-and-upload-read-files" title="Permalink to this headline">¶</a></h3>
<p>Before starting submission of reads and experiment data, you need to prepare and upload read files to your own directory in the Webin file upload area. We assume that you have paired-end reads in FASTQ format, like the example files from the <a class="reference external" href="https://www.mothur.org/wiki/MiSeq_SOP">mothur MiSeq SOP</a> that we use below, but please refer to <a class="reference external" href="https://ena-docs.readthedocs.io/en/latest/fileprep/reads.html">ENA’s guidelines on file formats</a> for other options:</p>
<div class="section" id="step-4a-compress-files-and-calculate-checksums">
<h4>Step 4a: Compress files and calculate checksums<a class="headerlink" href="#step-4a-compress-files-and-calculate-checksums" title="Permalink to this headline">¶</a></h4>
<p>On a Mac, open the Terminal app (in <em>Applications | Utilities</em>), and do the following:</p>
<div class="highlight-bash notranslate"><div class="highlight"><pre><span></span><span class="c1"># Go to read file directory</span>
<span class="c1"># Tip: type &#39;cd &#39; and then drag/drop folder from Finder into Terminal</span>
<span class="nb">cd</span> ~/your-read-file-dir

<span class="c1"># Compress read files, if they are uncompressed (edit file extension as needed)</span>
gzip -k *.fastq
</pre></div>
</div>
<p>To enable verification of file integrity after upload, calculate the md5 checksum of each (compressed) read file:</p>
<div class="highlight-bash notranslate"><div class="highlight"><pre><span></span><span class="c1"># Calculate and print md5 sums to tab-separated file (for easy cut-and-paste later)</span>
<span class="k">for</span> f <span class="k">in</span> *.gz<span class="p">;</span> <span class="k">do</span> md5 <span class="nv">$f</span> <span class="p">|</span> awk <span class="s1">&#39;{ gsub(/\(|\)/,&quot;&quot;); print $2&quot;\t&quot; $4 }&#39;</span><span class="p">;</span> <span class="k">done</span> &gt; md5sums.tsv
</pre></div>
</div>
<p>The resulting file should look similar to this:</p>
<table class="docutils align-default">
<colgroup>
<col style="width: 50%" />
<col style="width: 50%" />
</colgroup>
<tbody>
<tr class="row-odd"><td><p>F3D148_S214_L001_R1_001.fastq.gz</p></td>
<td><p>bb8f57f8a94050de3d8ad47d03477e68</p></td>
</tr>
<tr class="row-even"><td><p>F3D148_S214_L001_R2_001.fastq.gz</p></td>
<td><p>d71d368d056b1d4bcf0ea74628f4d63a</p></td>
</tr>
<tr class="row-odd"><td><p>F3D149_S215_L001_R1_001.fastq.gz</p></td>
<td><p>f76e0d9366998f7509a7aaca640cc842</p></td>
</tr>
<tr class="row-even"><td><p>F3D149_S215_L001_R2_001.fastq.gz</p></td>
<td><p>6c9acb1f743d984e07a9ce6b35b0fb6a</p></td>
</tr>
<tr class="row-odd"><td><p>F3D150_S216_L001_R1_001.fastq.gz</p></td>
<td><p>e869e2b2bff6cd6c5dbe8f1b1d22d5bc</p></td>
</tr>
<tr class="row-even"><td><p>F3D150_S216_L001_R2_001.fastq.gz</p></td>
<td><p>f3c6d5a3dae027187ff5f80100d37482</p></td>
</tr>
</tbody>
</table>
<p>The <em>md5sum</em> command should work similarly on a Linux machine, but Windows users may need to install some application to compress files to *.gz format, and do a quick web search for <em>how to generate MD5 checksum for file on Windows</em>.</p>
</div>
<div class="section" id="step-4b-upload-read-files-to-ena">
<h4>Step 4b: Upload read files to ENA<a class="headerlink" href="#step-4b-upload-read-files-to-ena" title="Permalink to this headline">¶</a></h4>
<p>You can now upload your sequence reads to ENA’s transit area, but remember to keep local copies of your data. There are several FTP (File Transfer Protocol) clients available for file transfer. We describe one alternative (<a class="reference external" href="https://formulae.brew.sh/formula/lftp">lftp</a>) for Mac users, but please refer to <a class="reference external" href="https://ena-docs.readthedocs.io/en/latest/fileprep/upload.html">ENA’s guidelines on file upload</a> for other options:</p>
<div class="highlight-bash notranslate"><div class="highlight"><pre><span></span><span class="c1"># Connect to FTP server [replace X:s, and provide password when prompted]</span>
lftp webin2.ebi.ac.uk -u Webin-XXXXX
<span class="c1"># Expected response: lftp Webin-XXXXX@webin2.ebi.ac.uk:~&gt;</span>

<span class="c1"># Transfer your read files</span>
mput ~/your-read-file-dir/*.fastq.gz
<span class="c1"># Expected response: ... Total x files transferred</span>

<span class="c1"># Disconnect from server</span>
bye
</pre></div>
</div>
</div>
</div>
<div class="section" id="step-5-submit-sequence-reads">
<h3>Step 5: Submit sequence reads<a class="headerlink" href="#step-5-submit-sequence-reads" title="Permalink to this headline">¶</a></h3>
<p>Once you have successfully uploaded your sequence reads, you need to associate them with already submitted sample metadata, as well as describe the sequencing <em>experiment</em> that produced the read files. In <a class="reference external" href="https://ena-docs.readthedocs.io/en/latest/general-guide/metadata.html">the ENA metadata model</a>, an <em>experiment</em> refers to a sequencing event, and contains information on e.g. library construction and instruments, whereas <em>runs</em> represent the read files resulting from an experiment.</p>
<div class="section" id="step-5a-download-the-experiment-run-template">
<h4>Step 5a: Download the experiment/run template<a class="headerlink" href="#step-5a-download-the-experiment-run-template" title="Permalink to this headline">¶</a></h4>
<p>Go to <em>Raw Reads (Experiments and Runs) | Submit Reads</em>, click <em>Download spreadsheet template for Read submission</em>, and select your file format. We assume you want to <em>Submit paired reads using two Fastq files</em>, here. Again, <em>Mandatory Fields</em> have been pre-selected for you, but you can also add <em>Optional Fields</em>. Have a look at permitted values before downloading your template (or simply return to this page later).</p>
<p>Your downloaded spreadsheet template should look something this:</p>
<table class="docutils align-default">
<colgroup>
<col style="width: 8%" />
<col style="width: 8%" />
<col style="width: 8%" />
<col style="width: 8%" />
<col style="width: 8%" />
<col style="width: 8%" />
<col style="width: 8%" />
<col style="width: 8%" />
<col style="width: 8%" />
<col style="width: 8%" />
<col style="width: 8%" />
<col style="width: 8%" />
</colgroup>
<tbody>
<tr class="row-odd"><td><p>FileType</p></td>
<td><p>fastq</p></td>
<td><p>Read submission file type</p></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr class="row-even"><td><p>sample</p></td>
<td><p>study</p></td>
<td><p>instrument_model</p></td>
<td><p>library_name</p></td>
<td><p>library_source</p></td>
<td><p>library_selection</p></td>
<td><p>library_strategy</p></td>
<td><p>library_layout</p></td>
<td><p>forward_file_name</p></td>
<td><p>forward_file_md5</p></td>
<td><p>reverse_file_name</p></td>
<td><p>reverse_file_md5</p></td>
</tr>
<tr class="row-odd"><td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
</tbody>
</table>
</div>
<div class="section" id="step-5b-describe-experiment-and-runs">
<h4>Step 5b: Describe experiment and runs<a class="headerlink" href="#step-5b-describe-experiment-and-runs" title="Permalink to this headline">¶</a></h4>
<p>In your downloaded template, you need to link each experiment/run row to an already submitted sample and study by adding the correct Sample and Study <em>Accession</em> values under the <em>sample</em> and <em>study</em> headers. Then, for metabarcoding data, you would typically use the following values to describe sequenced libraries (but see <a class="reference external" href="https://ena-docs.readthedocs.io/en/latest/reads/webin-cli.html">ENA’s guidelines on experiment metadata</a> for all available options):</p>
<ul class="simple">
<li><p><strong>library source:</strong> METAGENOMIC</p></li>
</ul>
<ul class="simple">
<li><p><strong>library selection:</strong> PCR</p></li>
</ul>
<ul class="simple">
<li><p><strong>library strategy</strong>: AMPLICON</p></li>
</ul>
<p>Also add the following metadata for read files:</p>
<ul class="simple">
<li><p><strong>forward [reverse]_file_name:</strong> If you put your read files directly into your account (in step 4b), you add filenames <em>only</em> here. Otherwise, enter paths including all subdirectories.</p></li>
</ul>
<ul class="simple">
<li><p><strong>forward [reverse]_file_md5:</strong> Paste the md5 checksums from Step 4a.</p></li>
</ul>
<p>Your spreadsheet should now look similar to this (but add as much optional metadata as possible):</p>
<table class="docutils align-default">
<colgroup>
<col style="width: 8%" />
<col style="width: 8%" />
<col style="width: 8%" />
<col style="width: 8%" />
<col style="width: 8%" />
<col style="width: 8%" />
<col style="width: 8%" />
<col style="width: 8%" />
<col style="width: 8%" />
<col style="width: 8%" />
<col style="width: 8%" />
<col style="width: 8%" />
</colgroup>
<tbody>
<tr class="row-odd"><td><p>FileType</p></td>
<td><p>fastq</p></td>
<td><p>Read submission file type</p></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
<td></td>
</tr>
<tr class="row-even"><td><p>sample</p></td>
<td><p>study</p></td>
<td><p>instrument_model</p></td>
<td><p>library_name</p></td>
<td><p>library_source</p></td>
<td><p>library_selection</p></td>
<td><p>library_strategy</p></td>
<td><p>library_layout</p></td>
<td><p>forward_file_name</p></td>
<td><p>forward_file_md5</p></td>
<td><p>reverse_file_name</p></td>
<td><p>reverse_file_md5</p></td>
</tr>
<tr class="row-odd"><td><p>ERS8208349</p></td>
<td><p>PRJEB48461</p></td>
<td><p>Illumina MiSeq</p></td>
<td><p>16S rRNA amplicon of Sample 1</p></td>
<td><p>METAGENOMIC</p></td>
<td><p>PCR</p></td>
<td><p>AMPLICON</p></td>
<td><p>PAIRED</p></td>
<td><p>F3D148_S214_L001_R1_001.fastq.gz</p></td>
<td><p>bb8f57f8a94050de3d8ad47d03477e68</p></td>
<td><p>F3D148_S214_L001_R2_001.fastq.gz</p></td>
<td><p>d71d368d056b1d4bcf0ea74628f4d63a</p></td>
</tr>
<tr class="row-even"><td><p>ERS8208348</p></td>
<td><p>PRJEB48461</p></td>
<td><p>Illumina MiSeq</p></td>
<td><p>16S rRNA amplicon of Sample 2</p></td>
<td><p>METAGENOMIC</p></td>
<td><p>PCR</p></td>
<td><p>AMPLICON</p></td>
<td><p>PAIRED</p></td>
<td><p>F3D149_S215_L001_R1_001.fastq.gz</p></td>
<td><p>f76e0d9366998f7509a7aaca640cc842</p></td>
<td><p>F3D149_S215_L001_R2_001.fastq.gz</p></td>
<td><p>6c9acb1f743d984e07a9ce6b35b0fb6a</p></td>
</tr>
<tr class="row-odd"><td><p>ERS8208347</p></td>
<td><p>PRJEB48461</p></td>
<td><p>Illumina MiSeq</p></td>
<td><p>16S rRNA amplicon of Sample 3</p></td>
<td><p>METAGENOMIC</p></td>
<td><p>PCR</p></td>
<td><p>AMPLICON</p></td>
<td><p>PAIRED</p></td>
<td><p>F3D150_S216_L001_R1_001.fastq.gz</p></td>
<td><p>e869e2b2bff6cd6c5dbe8f1b1d22d5bc</p></td>
<td><p>F3D150_S216_L001_R2_001.fastq.gz</p></td>
<td><p>f3c6d5a3dae027187ff5f80100d37482</p></td>
</tr>
</tbody>
</table>
<p>If so, go back to <em>Raw Reads (Experiments and Runs) | Submit reads</em>, click <em>Upload filled spreadsheet template for Read submission</em>, select your edited *.tsv file, and click <em>Submit Completed Spreadsheet</em>. If your submission is confirmed, you should now be able to se your submitted runs in the <em>Raw Reads (Experiments and Runs) | Run Files Report</em>.</p>
</div>
</div>
<div class="section" id="step-6-submit-to-production-service">
<h3>Step 6: Submit to production service<a class="headerlink" href="#step-6-submit-to-production-service" title="Permalink to this headline">¶</a></h3>
<p>After careful inspection of metadata, you can now repeat the process in the <a class="reference external" href="https://www.ebi.ac.uk/ena/submit/webin/login">production version of the Webin interface</a>.</p>
</div>
</div>
<div class="section" id="post-submission-editing">
<h2>Post-submission editing<a class="headerlink" href="#post-submission-editing" title="Permalink to this headline">¶</a></h2>
<p>In each row of the above mentioned <em>Reports</em>, you can click the box-arrow icon <img alt="action-icon" src="_images/action-icon.png" /> in the <em>Action</em> column to either show related items, e.g. see runs linked to a certain sample, or inspect/edit the underlying xml of a submitted item. But for more complex edits you likely need to <a class="reference external" href="mailto:datasubs&#37;&#52;&#48;ebi&#46;ac&#46;uk">contact ENA</a> and ask them to help you cancel and resubmit data.</p>
</div>
</div>


           </div>
           
          </div>
          <footer>
    <div class="rst-footer-buttons" role="navigation" aria-label="footer navigation">
        <a href="std-taxon-annotation.html" class="btn btn-neutral float-right" title="Taxonomic annotation of ASVs" accesskey="n" rel="next">Next <span class="fa fa-arrow-circle-right" aria-hidden="true"></span></a>
        <a href="index.html" class="btn btn-neutral float-left" title="Molecular biodiversity data in SBDI" accesskey="p" rel="prev"><span class="fa fa-arrow-circle-left" aria-hidden="true"></span> Previous</a>
    </div>

  <hr/>

  <div role="contentinfo">
    <p>
        &#169; Copyright 2019, Swedish Biodiversity Data Infrastructure.

    </p>
  </div>
    
    
    
    Built with <a href="https://www.sphinx-doc.org/">Sphinx</a> using a
    
    <a href="https://github.com/readthedocs/sphinx_rtd_theme">theme</a>
    
    provided by <a href="https://readthedocs.org">Read the Docs</a>. 

</footer>
        </div>
      </div>

    </section>

  </div>
  

  <script type="text/javascript">
      jQuery(function () {
          SphinxRtdTheme.Navigation.enable(true);
      });
  </script>

  
  
    
   

</body>
</html>